docs/tools/bench.md

```diff
@@ -46,6 +46,7 @@ The current benchmarks are:
 * `mat-mul`: Multiplies two `nxn` matrices using the [naive][matmul] matrix multiplication algorithm. The matrices are randomly generated using a [linear congruential generator][rng].
 * `max-subarray`: solution to the classic Maximum Subarray problem.
 * `mod_inv`: Calculates the [modular inverse][modinv] of `n` under to a prime modulus p.
+* `mod_pow`: Performs [modular exponentiation][modpow] using the right-to-left binary method as described in the Wikipedia article.
```

Comment:
```suggestion
* `mod_pow`: Performs [modular exponentiation][modpow] using the right-to-left binary method.
```

This already links to the Wikipedia article, so no need to say it again. :smiley: